lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -404,6 +414,79 @@ spec = do
         r <- joinStakePool ctx (p ^. #id) (w, "Secure Passprase")
         expectResponseCode HTTP.status404 r
 
+    it "STAKE_POOLS_ESTIMATE_FEE_01 - \
+        \wallet with [fee - 1] => cannot estimate fee" $ \ctx -> do
+        putStrLn "hello"
+        (_, p:_) <- eventually $
+            unsafeRequest @[ApiStakePool] ctx listStakePoolsEp Empty
+        fee <- do
+            w <- fixtureWallet ctx
+            (_, r) <- joinStakePoolFee ctx (p ^. #id) w
+            print r
+            return $ either (error . show) (getQuantity . amount) r
```

Comment:
**[suggestion]**

Re-write as: 

```hs
getFromResponse amount <$> joinStakePoolFee ctx (p ^. #id) w
```
**Why?**

The intent is clearer and the overall code is more concise.